### ISSUE TITLE
Description of `endpoints` in 4.1 implies some places may be JSON-LD, unspecified others may not

### ISSUE THREAD

**Issue by jernst** [w3c-social/activitypump#410]
The description of `endpoints` says:

> This mapping may be nested inside the actor document as the value or may be a link to a JSON-LD document

This indirectly appears to imply that in other places, links to JSON-LD documents are not permitted, otherwise why pointing it out? But there is nothing in the spec that identifies where links to JSON-LD documents may not be used.

**Comment by nightpool** [comment]
No, it does not imply that. You are reading too much into an informative note.

**Comment by evanp** [comment]
> why pointing it out? 

The `endpoints` collection is much more like a name/value property set than other ActivityPub objects. I can only think of a few other elements, like `streams` that are more about having a complex property value than an independent `Object`. So, the reason this is called out is that it isn't an `Object` per se, probably?, but is possible to define in a separate document.

I'd like to add a Primer page that talks more about `endpoints` and possibly discourages this behaviour. Not a lot of client implementations seem to expect it.
